include/boost/url/const_string.hpp

```diff
@@ -42,10 +42,43 @@ namespace urls {
 */
 class const_string : public string_view
 {
+public:
+    /** Size of the string buffer
```

Comment:
This is misleading because a user could think it means the "current" size. You could say "small buffer size" or "built-in capacity" or... idk, @pdimov ?